### PR TITLE
Adding current gpa,credit hours info and notes on SQ

### LINKED ISSUE #7: Document the schedule query approach with Bigquery
For Athletic Dept project we have scheduled queries instead of Matview now, That change we as made in order to address a performance issue on left join w.r.t to Postgres and now these tables live on BigQuery end
https://docs.google.com/document/d/1jS4Sg_LT-8M_ix7GbMuloWtZp4GplcC5vtb6WGjN4JA/edit
https://cloud.google.com/bigquery/docs/scheduling-queries

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #7 #11 #12

The current term logic to play around 

```
WITH
 term_info AS (
 SELECT
   academic_term_id,
   name AS term_name,
   at2.term_begin_date,
   at2.term_end_date,
   DATE_ADD(at2.term_end_date, INTERVAL 2 WEEK) as extended_date
 FROM
   `udp-umich-prod.context_store_entity.academic_term` at2
 WHERE at2.term_year>2022 order by term_begin_date),
current_term_logic as (
select * From term_info ats where 
DATE('2024-05-07') < DATE_ADD(ats.term_end_date, INTERVAL 2 WEEK)
AND DATE('2024-05-07') >= ats.term_begin_date )
select * from current_term_logic
```

### CODE REVIEW

**Review by zqian — APPROVED** [review]
The sql changes looks good to me. I left a comment for minor change.

**Comment by zqian** [comment]
@pushyamig Is this still a draft PR, or is it ready for review?

**Comment by pushyamig** [comment]
it's still a draft, since Hema has to confirm this change looks good. I am not anticipating things will change but I will wait a bit.

File: README.md [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
-## Current Term Courses Info
-1. We fetch current term courses information, student assignment, grades and background information from UDP. All the SQL quries associated from that are in `current_term_courses`
-2. Athletic dept is the main user of this view, which are Materialized views.
-3. MatViews are created on the BigQuery Context store and refreshed 1PM EST
+## Creating Scheduled Queries
+1. Go to BiqQuery Console for [scheduled queries](https://console.cloud.google.com/bigquery/scheduled-queries?project=udp-umich-prod) and click on `+ Create scheduled query in editor`
+2. Enter your query in the Editor and click `scheduled` button
+3. Add following in the pop-up window
+   1. Details and schedule: `datamart_name__name_you_want_to_give`. For example: `athletic_department_views__v1_student_current_term_course`
+   2. Schedule option: `days` and at `18:00`. you can added any number. Don't need to do anything for `Start now` and `End never`
```
[the remark this answers is not in the document]
Comment (pushyamig) [reply README.md]: Done

**Comment by pushyamig** [comment]
This is ready for review.

File: README.md [hunk 2]
```diff
@@ -2,13 +2,25 @@
 
 Common queries for Canvas data
 
-## Current Term Courses Info
-1. We fetch current term courses information, student assignment, grades and background information from UDP. All the SQL quries associated from that are in `current_term_courses`
-2. Athletic dept is the main user of this view, which are Materialized views.
-3. MatViews are created on the BigQuery Context store and refreshed 1PM EST
+## Creating Scheduled Queries
+1. Go to BiqQuery Console for [scheduled queries](https://console.cloud.google.com/bigquery/scheduled-queries?project=udp-umich-prod) and click on `+ Create scheduled query in editor`
+2. Enter your query in the Editor and click `scheduled` button
```
Comment (zqian) [inline README.md:7]: The button name is `SCHEDULE`

File: README.md [hunk 3]
```diff
@@ -2,13 +2,25 @@
...
+   3. Destination for Query result: 
+      1. Check the checkbox `Set a destination table for query results`
+      2. Choose the data set name `athletic_department_views`
+      3. Give a table name for example: `v1_student_current_term_course`
+4. Destination table write preferance: `Overwrite table`
 
-## Auto-Refreshed Materialized Views
-Details are here: https://resources.unizin.org/display/UDP/UDP+Context+store
+## Updating a Scheduled queries
+1. There is no update of scheduled query it's always Delete and create a new table. This limitation is from BQ side
```
Comment (zqian) [inline README.md:18]: Adding a break/dash character into the sentence:

There is no update of scheduled query - it's always Delete and create a new table

**Review by zqian — APPROVED** [review]
Suggested minor text changes

File: README.md (hunk 2 above)
Comment (pushyamig) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (pushyamig) [reply README.md]: Done

**Pull request merged**